keyboards/hs60/v2/info.json

```diff
@@ -1,6 +1,6 @@
 {
-  "keyboard_name": "clueboard/60",
-  "maintainer": "skullydazed",
+  "keyboard_name": "HS60v2",
+  "maintainer": "qmk",
```

Comment:
Could we change this to: `"maintainer": "yiancar",` please?:)

Reply:
Done, now I know who to come to with my questions :)